--- src/raw_decoder.c       Fri Jul 21 06:17:38 2006
+++ src/raw_decoder.c       Fri Jul 21 06:17:44 2006
@@ -202,7 +202,7 @@ _vbi_service_table [] = {
                { 21, 0 },
                { 21, 0 },
                10500, 1006976, 503488, /* 32 x FH */
-               0x00005551, 0x7FF, 14, 2, 2 * 8, VBI_MODULATION_NRZ_LSB,
+               0x03, 0x0F, 4, 0, 2 * 8, VBI_MODULATION_NRZ_LSB,
                /* I've seen CC signals on other lines and there's no
                   way to distinguish from the transmitted data. */
                _VBI_SP_FIELD_NUM | _VBI_SP_LINE_NUM,
@@ -213,7 +213,7 @@ _vbi_service_table [] = {
                { 0, 284 },
                { 0, 284 },
                10500, 1006976, 503488, /* 32 x FH */
-               0x00005551, 0x7FF, 14, 2, 2 * 8, VBI_MODULATION_NRZ_LSB,
+               0x03, 0x0F, 4, 0, 2 * 8, VBI_MODULATION_NRZ_LSB,
                _VBI_SP_FIELD_NUM | _VBI_SP_LINE_NUM,
        }, {
                VBI_SLICED_2xCAPTION_525, /* NOT CONFIRMED */ 
